src/FixedDataTableCell.js

```diff
@@ -125,7 +125,10 @@ class FixedDataTableCell extends React.Component {
     return false;
   }
 
-  componentWillReceiveProps(props) {
+  componentDidUpdate(prevProps) {
+    if (prevProps === this.props) return;
```

Comment:
Is this only true when the component gets rerendered due to a state change within itself? 
Also can you change reformat this to:
```js
if (prevProps === this.props) {
  return;
}
```

In the long term, can we skip the extra rerender through use of `getDerivedStateFromProps` ? @wcjordan

Reply:
Also, since `getDerivedStateFromProps` was added in React v16.3, will this mean we'll deprecate every react version under v16.3.

Reply:
Yuck, we don't want to deprecate support for everything under 16.3.  @pradeepnschrodinger could you ticket out the idea of using getDerivedStateFromProps and we can discuss when may make sense to do that on our roadmap?

Reply:
Created #512

Reply:
Great, thanks